setup.py

```diff
@@ -201,13 +202,17 @@ def setup_package():
         cmake_collect_dirs=NRN_COLLECT_DIRS,
         cmake_flags=[
             '-DNRN_ENABLE_CORENEURON=OFF',
-            '-DNRN_ENABLE_INTERVIEWS=OFF',
+            '-DNRN_ENABLE_INTERVIEWS=ON',
+            '-DIV_ENABLE_X11_DYNAMIC=ON',
             '-DNRN_ENABLE_RX3D=OFF',  # Never build within CMake
-            '-DNRN_ENABLE_MPI=OFF',
+            '-DNRN_ENABLE_MPI=ON',
+            '-DNRN_ENABLE_MPI_DYNAMIC=ON',
             '-DNRN_ENABLE_PYTHON_DYNAMIC=ON',
             '-DNRN_ENABLE_MODULE_INSTALL=OFF',
             '-DNRN_USE_REL_RPATH=ON',
             '-DLINK_AGAINST_PYTHON=OFF',
+            '-DLINK_AGAINST_MPI=OFF',
+            '-DCMAKE_PREFIX_PATH=/opt/ncurses', #TODO : fix this custom path (Fernando)
```

Comment:
Is it too bad to ask people to do `install libncurses5`? I see this as an expected problem that all wheels in the world potentially have - so people know how to solve it, and soon, with Michael, we probably can drop the need for ncurses altogether.
By what I read, `libncursesw` (from manylinux whitelist) and `libncurses` always go together, as the first is simply the "wide symbol" version. Why Auditwheel creates a copy is a bit of a mystery to me. Eventually with `ldopen` it could work, as the `soname` should still be the original